source/quick-reference.txt

```diff
@@ -0,0 +1,561 @@
+          :copyable: true
+
+          .. input::
+             :language: csharp
+
+             var filter = Builders<Restaurant>.Filter
+               .Eq(restaurant => restaurant.Cuisine, "Afghan";
+
+             var cursor = _restaurantsCollection.Find(filter).ToCursor(); 
+             cursor.MoveNext();
+             foreach (Restaurant r in cursor.Current)
+             {
+                 WriteLine(r.ToJson());
+             }   
+
+          .. output::
+             :language: json
+
+             [
+               { name: "Afghan Kebab House", ... },
+               { name: "Khyber Pass", ... },
+               ...
+             ]
+
+   * - | **Count Documents**
+       |
+       | `API Documentation <{+api-root+}/Overload_MongoDB_Driver_IMongoCollection_1_CountDocuments.htm>`__
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: csharp
+             
+             var filter = Builders<Restaurant>.Filter
+               .Eq(restaurant => restaurant.Cuisine, "Pizza");
+
+             return _restaurantsCollection.Find(filter).CountDocuments();
+
+          .. output::
+             :language: none
+
+             1163
+
+   * - | **List the Distinct Documents or Field Values**
+       |
+       | `API Documentation <+api-root+}/Overload_MongoDB_Driver_MongoCollection_Distinct.htm>`__
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: csharp
+            
+             var filter = Builders<Restaurant>.Filter.Empty;
+             return _restaurantsCollection
+               .Distinct(new StringFieldDefinition<Restaurant, String>("cuisine"), filter);
+ 
+          .. output::
+             :language: json
+
+             [ Afghan, African, American, Armenian, Asian, ... ]
+
+   * - | **Limit the Number of Documents Retrieved**
+       |
+       | `API Documentation <+api-root+}/P_MongoDB_Driver_Core_Operations_FindOperation_1_Limit.htm>`__
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: csharp
+
+             var filter = Builders<Restaurant>.Filter
+               .Eq(restaurant => restaurant.Cuisine, "Asian");
+             return _restaurantsCollection.Find(filter).Limit(3).ToList();
+
+          .. output::
+             :language: json
+
+             [
+               { name: "China Grill", ... },
+               { name: "Indo Chine", ... },
+               { name: "Le Colonial", ... }
+             ]
+
+   * - | **Skip Retrieved Documents**
+       |
+       | `API Documentation <{+api-root+}P_MongoDB_Driver_Core_Operations_FindOperation_1_Skip.htm>`__
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: csharp
+
+             var filter = Builders<Restaurant>.Filter
+               .Eq(restaurant => restaurant.Cuisine, "Asian");
+             return _restaurantsCollection.Find(filter).Skip(2).ToList();
```

Comment:
Space between statements for readability (and consistency with other code blocks)

```suggestion
               .Eq(restaurant => restaurant.Cuisine, "Asian");
               
             return _restaurantsCollection.Find(filter).Skip(2).ToList();
```